cert/deployed/files.sls

```diff
@@ -0,0 +1,81 @@
+# -*- mode: salt; coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_package_install = tplroot ~ '.package.install' %}
+{%- from tplroot ~ "/map.jinja" import mapdata without context %}
+
+# Deploy certificates
+# Place all files in a files_roots/cert, e.g. /srv/salt/files/cert/
+
+# Make sure we only run update-ca-certificates if certificates were managed
+{% set vals = { 'managed_certs': False } %}
```

Comment:
If I understand correctly, you set `managed_certs=True` if you have at least one item in `mapdata.get('certlist')`.
I think you can use directly the value of `mapdata.get('certlist', None)` or `mapdata.get('certlist', {}` directly, which would be simplier.

Reply:
Thanks @daks, this code was here previously, I just wanted to refactor it to `template-formula` layout with minimal changes.

I agree that this construct is somewhat strange but instead of using such semaphore I'll change that to the more salty [onchannges](https://docs.saltproject.io/en/latest/ref/states/requisites.html#onchanges) in future commits.

Reply:
When I speak about `onchanges`, I mean on the whole `.sls` instead of using `onchanges_in` like done today.